packages/contracts-bedrock/interfaces/L2/IL2ToL2CrossDomainMessenger.sol

```diff
@@ -45,19 +45,28 @@ interface IL2ToL2CrossDomainMessenger {
     /// @notice Thrown when attempting to use a chain ID that is not in the dependency set.
     error InvalidChainId();
 
+    /// @notice Thrown when relaying a message from an address different than the specified entrypoint.
+    error InvalidEntrypoint();
+
     /// @notice Emitted whenever a message is sent to a destination
     /// @param destination  Chain ID of the destination chain.
     /// @param target       Target contract or wallet address.
-    /// @param messageNonce Nonce associated with the messsage sent
+    /// @param messageNonce Nonce associated with the message sent
     /// @param sender       Address initiating this message call
     /// @param message      Message payload to call target with.
+    /// @param entrypoint   Address of the entrypoint on the destination chain.
     event SentMessage(
-        uint256 indexed destination, address indexed target, uint256 indexed messageNonce, address sender, bytes message
+        uint256 indexed destination,
+        address indexed target,
+        uint256 indexed messageNonce,
+        address sender,
+        bytes message,
```

Comment:
small nit: `entrypoint` prior to `message` such that all fixed length members come first. Makes it a bit easier to decode

Reply:
Hamdi! Good catch, we’ve already made the changes. Thanks for the feedback!